app/forms/state_file/ny_primary_state_id_form.rb

```diff
@@ -0,0 +1,67 @@
+module StateFile
+  class NyPrimaryStateIdForm < QuestionsForm
+    include DateHelper
+    set_attributes_for :state_id, :id_type, :id_number, :state, :first_three_doc_num
+
+    set_attributes_for :dates,
+                       :issue_date_day,
+                       :issue_date_month,
+                       :issue_date_year,
+                       :expiration_date_day,
+                       :expiration_date_month,
+                       :expiration_date_year
+
+    # maybe the validation shouldn't be so harsh? Do we know what the constraints are
```

Comment:
I will remove these comments before merging, I have floated these questions to Courtney. Might be something we get to later though.